src/autobots/conn/openai/openai_chat/openai_chat.py

```
@@ -13,9 +13,17 @@ def __init__(self, openai_client: AsyncOpenAI):
 
     @retry(exceptions=Exception, tries=3, delay=30)
     async def chat(self, chat_req: ChatReq) -> ChatCompletion | AsyncStream[ChatCompletionChunk] | None:
+        # model vision is resulting in error because of these 6 extra params
+        if chat_req.model.__contains__("-vision-"):
+            chat_req.logit_bias = None
```

Thanks for fixing this so quickly. Should we throw an error instead, or maybe just log an error, if any of these is not None? This could become tricky to debug otherwise in situations where people are trying to use any of these parameters with the vision model